pkg/apis/core/v1beta1/helper/errors.go

```diff
@@ -223,6 +223,23 @@ type WrappedLastErrors struct {
 	LastErrors  []gardencorev1beta1.LastError
 }
 
+// DeprecatedNewWrappedLastErrors returns an error
+func DeprecatedNewWrappedLastErrors(description string, err error) *WrappedLastErrors {
+	var lastErrors []gardencorev1beta1.LastError
+
+	for _, partError := range utilerrors.Errors(err) {
+		lastErrors = append(lastErrors, *LastErrorWithTaskID(
+			partError.Error(),
+			utilerrors.GetID(partError),
+			DeprecatedDetermineErrorCodes(partError)...))
+	}
+
+	return &WrappedLastErrors{
+		Description: description,
+		LastErrors:  lastErrors,
+	}
+}
+
 // NewWrappedLastErrors returns an error
```

Comment:
Can we improve the doc string? like, returns a LastError with the error and error codes or something like that?